### ISSUE TITLE
Meshes not visible when using dae files that include the transparency tag.

### ISSUE THREAD

**Issue by Levi-Armstrong** [ignitionrobotics/ign-rendering#89]
I recently ran into the issue described [here](https://answers.gazebosim.org//question/20068/collada-model-is-not-visible-in-gazebo-89-if-transparent-a_one/). I had a mesh that was not showing up, so I opened up the dae file an remove the transparency tag shown in the link and now the mesh shows up.

**Comment by chapulina** [comment]
@Levi-Armstrong , are you using Ogre 1 or 2? Or is the issue on both?

**Comment by Levi-Armstrong** [comment]
I was using Ogre1, Ubuntu 18.04 and ignition-rendering3.

**Comment by iche033** [comment]
The collada loader [transparency fix](https://osrf-migration.github.io/ignition-gh-pages/#!/ignitionrobotics/ign-rendering/pull-requests/2616) should be ported to ign-common. Could you post the `<transparency>` and `<transparent`> tags from your collada file?

**Comment by Levi-Armstrong** [comment]
In my case it only had the transparent tag and did not have the transparency tag. I removed the lines below and it showed up, but the comments in the link also suggested adding the transparency  tag setting the alpha to one also works.
```
<transparent opaque="A_ONE">
     <color>1 1 1 1</color>
 </transparent>
```

**Comment by iche033** [comment]
made a fix in ignitionrobotics/ign-common#68

**Comment by chapulina** [comment]
`ign-common` PR merged, the fix will be available on the next release.
